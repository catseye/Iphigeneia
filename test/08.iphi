(* Test 'if ... then ... else' expression with negative result *)

var a in var b in var c in
begin
    a := 10;
    b := 2;
    c := if a > 20 then a - b else a / b
end
