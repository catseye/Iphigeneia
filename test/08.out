c=5
b=2
a=10
